### PR TITLE
items-per-row: avoid changing layout with default settings

### LINKED ISSUE #3293: items-per-row changes row height on default settings
On explore page, 4 items per row changes the height of the projects:

https://user-images.githubusercontent.com/17484114/129493825-cee9d26b-db93-4a06-a7f9-323c525868af.mp4

### PR DESCRIPTION
**Pull request by mxmou**
Resolves #3293

### Changes

Modifies the CSS of the `items-per-row` addon to avoid changing the height of rows on Explore, search, and studio pages.

### Reason for changes

If the selected number of items is the same as Scratch's default, the addon shouldn't make any changes.

### Tests

Tested on Edge and Firefox.

### CODE REVIEW

**Review by WorldLanguages — APPROVED** [review]
This can be merged

**Comment by mxmou** [comment]
@WorldLanguages I can't merge without two approvals.

**Comment by Samq64** [comment]
There still seems to be a little bit of a layout shift, but it's not as bad as before.

[Screencast from 2023-02-04 09-42-55.webm](https://user-images.githubusercontent.com/81489795/216773565-b2c0810b-807a-4143-a78d-fe0635bd2d64.webm)

**Review by TheColaber — APPROVED** [review]

**Pull request merged**
